package TestingSuite;

public class Exceptions {

	public static void main(String[] args) {
		System.out.println("test");
		System.out.println("test1");
		System.out.println("test2");
		System.out.println("test3");
		/*int x = 10;
		int y = 0;
		//int c= x/y;
		
		try {
			int c= x/y;
		}
		catch (Exception e)
		{
			System.out.println(e);
		}
		*/
		
		System.out.println("test4");
		System.out.println("test5");

	}

}
